src/main/java/core/basesyntax/operationstrategy/OperationStrategy.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.operationstrategy;
+
+import core.basesyntax.model.OperationFruitDto;
+import java.util.HashMap;
+import java.util.Map;
+
+public class OperationStrategy implements OperationService {
+    public static final Map<String, OperationService> mapOperation = new HashMap<>();
```

Comment:
you are confusing service and strategy

let's have Strategy class with the map and the method that returns the correct handler

and then let's have OperationHandler interface with method apply
and its implementations: AddOperationHandler implements OperationHandler, and so on

in `main()` call the strategy method, get the handler, and call `apply()`